vars/govuk.groovy

```diff
@@ -109,7 +109,7 @@ def buildProject(Map options = [:]) {
         "build" : { nonDockerBuildTasks(options, jobName, repoName) },
         "docker" : { dockerBuildTasks(options, jobName) }
       )
-    } else {
+    } else if (env.BRANCH_NAME == "master" || !options.migratedToGithubActions) {
```

Comment:
```
build my stuff if branch name is master
build my stuff if on any branch and we've not migrated to github actions
```

Excuse the pseudocode, but I'm trying to understand... So the only way this would never build at all is if there's not a Jenkinsfile (if we fully migrate to Actions, in the future).

Reply:
Yup, that's right 👍. Maybe it would be simpler if I did it in a separate `if` branch. What do you think?

Reply:
I think so, please!

Reply:
Sure, updated 👍